### PR TITLE
feat(ui-contract-editor):string variable can be made empty(#150)

### LINKED ISSUE #150: empty string variable
# Feature Request 🛍️
Because we removed the “”, it looks like a user can no longer delete the first character of a string variable by pressing backspace. This makes it really annoying to change the text of a string variable (unless you happen to want to change the variable to a different string that has the same first letter). Strangely the user **can** delete the first character by highlighting all characters in a variable and then pressing backspace.
https://www.loom.com/share/6f3b65e5684d47568bb8172c2bbe5638

## Possible Solution
Allow all characters to be removed from a variable consistently.

### PR DESCRIPTION
**Pull request by sanketshevkar**
Signed-off-by: User <shevkar.sanket@gmail.com>

# Closes #150 

### Changes

- Code commented in isEditableVariable in withVariables.js

### Screenshots or Video

[https://www.loom.com/share/91e95c6d657e40d48d41cf41e2da0812](url)

### Related Issues
- Issue #150

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`
- [ ] Manual accessibility test performed
    - [ ] Keyboard-only access, including forms
    - [ ] Contrast at least WCAG Level A
    - [ ] Appropriate labels, alt text, and instructions

### CODE REVIEW

File: packages/ui-contract-editor/src/ContractEditor/plugins/withVariables.js [hunk 1]
```diff
@@ -57,16 +57,15 @@ export const isEditableVariable = (lockText, editor, event) => {
...
   if (editable) {
     if (atEnd(editor) && event.inputType === 'deleteContentForward') {
       return false;
     }
     if (event.inputType === 'deleteContentBackward') {
       // Do not allow user to delete variable if only 1 char left
-      if (textLength === 1) {
-        return false;
-      }
+      // if (textLength === 1) {
```
Comment (dselman) [inline packages/ui-contract-editor/src/ContractEditor/plugins/withVariables.js:66]: Please remove the commented code and clean up

**Review by dselman — CHANGES_REQUESTED** [review]

File: packages/ui-contract-editor/src/ContractEditor/plugins/withVariables.js (hunk 1 above)
Comment (sanketshevkar) [reply packages/ui-contract-editor/src/ContractEditor/plugins/withVariables.js:66]: Sure, will do that.

Comment (sanketshevkar) [reply packages/ui-contract-editor/src/ContractEditor/plugins/withVariables.js]: Done.

**Review by dselman — APPROVED** [review]
Nice!

**Review by DianaLease — APPROVED** [review]

**Pull request merged**